cmd/server/cmd/BUILD

```diff
@@ -31,7 +31,6 @@ go_library(
         "@com_github_grpcecosystem_opentracing//go/otgrpc:go_default_library",
         "@com_github_grpcecosystem_prometheus//:go_default_library",
         "@com_github_istio_api//:mixer/v1",
-        "@com_github_istio_api//:mixer/v1/config",
```

Comment:
Does all the files from where you have removed "@com_github_istio_api//:mixer/v1/config"', do we need to reference mixer/v1/config_fixed ?